From a9e87d7423b45ce23865d3b6668185daee0f4ab7 Mon Sep 17 00:00:00 2001
From: Maulik Jodhani <maulik.jodhani@xilinx.com>
Date: Fri, 17 Mar 2017 07:19:29 -0700
Subject: [PATCH 038/532] Documentation: DT: macb: update for partial store and
 forward

commit 50c6be60c341ecf2974e56849256eb8b9d72a31b from
https://github.com/Xilinx/linux-xlnx.git

Update dt-binding document for partial store and forward mode
support in macb ethernet driver.

Signed-off-by: Maulik Jodhani <maulik.jodhani@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 Documentation/devicetree/bindings/net/macb.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Documentation/devicetree/bindings/net/macb.txt b/Documentation/devicetree/bindings/net/macb.txt
index af2756b..f722b57 100644
--- a/Documentation/devicetree/bindings/net/macb.txt
+++ b/Documentation/devicetree/bindings/net/macb.txt
@@ -31,6 +31,9 @@ Optional properties for PHY child node:
 - phy-reset-gpio: GPIO number for phy reset.
 - phy-reset-duration: Reset duration in milliseconds.
 - phy-reset-active-low: Boolean flag to indicate if phy reset is active low.
+- rx-watermark: Set watermark value for pbuf_rxcutthru reg and enable
+  rx partial store and forward, only when compatible = "cdns,zynqmp-gem".
+  Value should be less than 0xFFF.
 
 Examples:
 
@@ -38,6 +41,7 @@ Examples:
 		compatible = "cdns,at32ap7000-macb";
 		reg = <0xfffc4000 0x4000>;
 		interrupts = <21>;
+		rx-watermark = /bits/ 16 <0x44>;
 		phy-mode = "rmii";
 		local-mac-address = [3a 0e 03 04 05 06];
 		clock-names = "pclk", "hclk", "tx_clk";
-- 
2.7.4

